### DISCUSSION TITLE
Error when completing file with no results

### DISCUSSION THREAD

**Discussion by kevinji** [minad/vertico#403] [Q&A] [answered]
In the "Find file" minibuffer, I'm running into the following error while trying to complete a file with no results:
```
[Error in pre-command-hook (vertico--prepare): (error "Invalid completion style +vertico-basic-remote")]
```

Specifically, upon entering the keystroke that would end up with an empty list of files, this error shows up and the minibuffer no longer works.

Is there anything in my Vertico config that's causing this error? I was mostly basing this off the config in the [Configure Vertico per command or completion category](https://github.com/minad/vertico#configure-vertico-per-command-or-completion-category) section:
```elisp
(vertico-multiform-mode)

(defun sort-directories-first (files)
  (setq files (vertico-sort-alpha files))
  (nconc
    (seq-filter (lambda (x) (string-suffix-p "/" x)) files)
    (seq-remove (lambda (x) (string-suffix-p "/" x)) files)))

(setq vertico-multiform-categories
  '((file (vertico-sort-function . sort-directories-first))))
```

**Comment by minad** [comment, marked as answer]
The error is `(error "Invalid completion style +vertico-basic-remote")`, so you use this undefined completion style somewhere.

**Reply by kevinji** [reply]
That's a good catch, I think my config was a red herring that led me astray because the issue seemed to have intermittently gone away when I removed my config changes. Going to reach out with the Doom Emacs project instead.

**Comment by pivaldi** [comment]
See https://github.com/doomemacs/doomemacs/issues/7491
